JiraReporter/RestApiRequests.cs

```diff
@@ -11,82 +11,84 @@ namespace JiraReporter
 {
     class RestApiRequests
     {
-        public static Timesheet GetTimesheet(SourceControlLogReporter.Model.Policy policy, DateTime startDate, DateTime endDate)
+        public static string ResolveRequest(SourceControlLogReporter.Model.Policy policy, RestRequest request)
```

Comment:
@vladichimescu 
Mai bine faci o metoda generica `public static T ResolveRequest<T>(Policy, RestRequest)` care sa includa si deserializarea. 

De asemenea, metoda asta cu username/pass si cea cu jwt ar trebui sa fie combinate si decizia sa se ia in cadrul metodei, nu sa fie 2 metode diferite.